exercises/practice/camicia/lib/camicia.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Camicia do
+  @doc """
+    Simulate a card game between two players.
+    Each player has a deck of cards represented as a list of strings.
+    Returns a tuple with the result of the game:
+    - `{:finished, cards, tricks}` if the game finishes with a winner
+    - `{:loop, cards, tricks}` if the game enters a loop
+    `cards` is the number of cards played.
+    `tricks` is the number of central piles collected.
+
+    ## Examples
+
+      iex> Camicia.simulate(["2"], ["3"])
+      {:finished, 2, 1}
+
+      iex> Camicia.simulate(["J", "2", "3"], ["4", "J", "5"])
+      {:loop, 8, 3}
+  """
+
+  @spec simulate(list, list) :: {:finished | :loop, non_neg_integer(), non_neg_integer()}
```

Comment:
We could be a bit more specific:
```suggestion
  @spec simulate(list(String.t()), list(String.t()) :: {:finished | :loop, non_neg_integer(), non_neg_integer()}
```